snapshot/spec/runner_spec.rb

```diff
@@ -7,5 +7,27 @@
         expect(helper_version).to match(/^SnapshotHelperVersion \[\d.\d\]$/)
       end
     end
+
+    describe 'output_simulator_logs' do
+      before(:each) do
+        allow(FileUtils).to receive(:cp).with(anything, anything) do
+          # nothing
+        end
```

Comment:
please remove the block.
`allow(FileUtils).to receive(:cp).with(anything, anything)` should do it.